openapi/schemas/processes-core/statusInfo.yaml

```diff
@@ -1,17 +1,19 @@
 type: object
 required:
-   - jobID
+   - id
    - status
    - type
 properties:
+   id:
+      type: string
    processID:
       type: string
    type:
       type: string
-      enum:
+      example:
         - process
-   jobID:
-      type: string
+        - wps
```

Comment:
wps?

Reply:
Since not `enum` but `example`, it doesn't really matter. 
Some servers (CRIM's for example) do support WPS this way.